src/Components/CloudAccounts/CloudAccountsUtils.tsx

```diff
@@ -0,0 +1,20 @@
+import { Provider } from './types';
+export const providerIdToProvider: Record<string, Provider> = {
+  aws: 'AWS',
+  azure: 'Azure',
+  gcp: 'Google Cloud',
+};
+
+export const providerToParam: Record<Provider, string> = {
+  AWS: 'aws',
+  Azure: 'azure',
+  'Google Cloud': 'google',
+} as const;
+
+export const PROVIDER_MAP = {
+  AWS: 'AWS',
+  GCE: 'Google Cloud',
+  MSAZ: 'Azure',
+} as const;
+
+export type ProviderKey = keyof typeof PROVIDER_MAP;
```

Comment:
Enhancement: Use clearer names for these constants - I can't tell from this file or from their names what they are actually mean